extensions/emmet/src/abbreviationActions.ts

```diff
@@ -16,6 +16,9 @@ const inlineElements = ['a', 'abbr', 'acronym', 'applet', 'b', 'basefont', 'bdo'
 	's', 'samp', 'select', 'small', 'span', 'strike', 'strong', 'sub', 'sup',
 	'textarea', 'tt', 'u', 'var'];
 
+let previousAbbreviation = '';
+let previousAbbreviationIndividualLines = '';
```

Comment:
You can just assume `individualLines = true` now, so I think just having `previousAbbreviation` is ok.